tests/test_modeling_common.py

```diff
@@ -211,8 +216,11 @@ def recursive_check(tuple_object, dict_object):
         model = self.model_class(**init_dict)
         model.to(torch_device)
         model.eval()
+        if isinstance(model, ModelMixin):
+            model._mps_warmup(inputs_dict['sample'].shape[0])
 
-        outputs_dict = model(**inputs_dict)
-        outputs_tuple = model(**inputs_dict, return_dict=False)
+        with torch.no_grad():
```

Comment:
I added `with torch.no_grad():` because computing gradients does not work well in `mps`. Is it important to do it in  the other devices and we can disable them for all these tests? /cc @patrickvonplaten @patil-suraj